src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CodeActionResolutionEndpoint.cs

```diff
@@ -48,7 +46,7 @@ internal class CodeActionResolutionEndpoint : IRazorCodeActionResolveHandler
             _csharpCodeActionResolvers = CreateResolverMap(csharpCodeActionResolvers);
         }
 
-        public async Task<CodeAction> Handle(CodeAction request, CancellationToken cancellationToken)
+        public async Task<CodeAction?> Handle(CodeActionBridge request, CancellationToken cancellationToken)
```

Comment:
Where does this ever return null?

EDIT: Oh, is this because of the interface? In that case, should we update any of the cases below to return null instead of returning `request`, or throwing?

Reply:
You were right the first time actually, the LSP spec says null is now allowed here, so I'll make the interface reflect that.